src/pages/blog/airship-update-may-2020.md

```diff
@@ -0,0 +1,142 @@
+---
+templateKey: blog-post
+title: Airship Update - May 2020
+author: Alexander Hughes
+date: 2020-05-15T12:00:00.000Z
+category: 
+  - label: Airship 2.0
+    id: category-C98iZYrE1
+---
+
+What follows is the Airship Community Update for the month of May 2020, brought to you by the Airship Technical
+Committee.<!-- more -->
+
+## **SPOTLIGHT ON ....**
+
+- Learn how Airship 2.0, targeted for delivery later this year, is solving NFVi challenges on the way to full 5G
+  deployment in this [*white paper*](https://www.airshipit.org/blog/airship2-whitepaper/) authored by AT&T's VP of
```

Comment:
Might read better as "VP of AT&T's Network Cloud"

Reply:
Good catch, thanks.